### ISSUE TITLE
let's add us to the main list

### ISSUE THREAD

**Issue by yegor256** [objectionary/eo-hamcrest#88]
Here: https://github.com/hamcrest/hamcrest.github.com

Let's submit a pull request there. Maybe they will accept (even though Kotlin is waiting since 2016 :)

**Comment by yegor256** [comment]
@Graur maybe you can try?

**Comment by Graur** [comment]
@yegor256 Yes, I started [a new pull request](https://github.com/hamcrest/hamcrest.github.com/pull/18) a couple month ago. Still no answer
